CriteoGoogleAdapter/CriteoGoogleAdapter.xcodeproj/project.pbxproj

```diff
@@ -80,6 +83,7 @@
 		3CF2926D760C675680C3BA20 /* Frameworks */ = {
 			isa = PBXGroup;
 			children = (
+				CF6AE25D24CEB22F00E00CDA /* CriteoPublisherSdk.framework */,
```

Comment:
I doubt you have to add that framework, if you use the workspace that should be automatic.

Reply:
The tests didn't pass for the AdMob adapter. I diff the AdMob/MoPub config to see were it comes from, and apparently this is a fix.

Reply:
Interesting, that is only because of the use of `NSProtocolFromString`, that class is discarded not being explicitly referenced otherwise.